### PR TITLE
Feature/movie page

### PR DESCRIPTION
**Pull request by nikitius20**

### CODE REVIEW

File: src/components/MovieCard.vue [hunk 1]
```diff
@@ -1,6 +1,9 @@
 <template>
-  <div class="col-lg-2 col-md-2 col-sm-3 col-5 m-3 p-0">
-    <div class="card border-light">
+  <div
+    class="movie-card col-lg-2 col-md-2 col-sm-3 col-5 m-3 p-0"
```
Comment (skss-1) [inline src/components/MovieCard.vue]: Because it's clickable, this element should have `cursor: pointer`

File: src/components/CreditsScroll.vue [hunk 2]
```diff
@@ -0,0 +1,73 @@
+<template>
+  <div class="credits my-3">
+    <div
+      v-for="actor in credits"
+      :key="actor.id"
+      class="card actor-card"
+    >
+      <img
```
Comment (skss-1) [inline src/components/CreditsScroll.vue:8]: Image should have the same size, even when it's not loaded
image

File: src/components/CreditsScroll.vue [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+    methods: {
+        imageUrl(profile_path) {
+            return `${posterPath}${profile_path}`
+        },
+    }
+}
+</script>
+
+<style scoped lang="scss">
+
```
Comment (skss-1) [inline src/components/CreditsScroll.vue]: Redundant empty line

File: src/components/CreditsScroll.vue [hunk 4]
```diff
@@ -0,0 +1,73 @@
+<template>
+  <div class="credits my-3">
```
Comment (skss-1) [inline src/components/CreditsScroll.vue]: Selector of the root element should be the same as component name.

File: src/views/MoviePageView.vue [hunk 5]
```diff
@@ -0,0 +1,153 @@
...
+    },
+    posterUrl() {
+      return `https://image.tmdb.org/t/p/original${this.item.poster_path}`
+    },
+    backgroundUrl() {
+      return `backgroundImage: linear-gradient(0deg, rgba(19, 21, 46, 0.7), rgba(19, 21, 46, 0.7)), url(https://image.tmdb.org/t/p/original${this.item.backdrop_path})`
+    },
+    releaseDate() {
+        const month= ['January','February','March','April','May','June','July','August','September','October','November','December']
+        let date=`${month[+this.item.release_date.slice(5,7) -1]} ${+this.item.release_date.slice(8,10)} ${this.item.release_date.slice(0,4)}`
```
Comment (skss-1) [inline src/views/MoviePageView.vue]: It's ok to have an array for months for some cases. But not ok to parse date from string.

Better to use `Date` object. You should parse and create the date at first: `const releaseDate = new Date(Date.parse(this.item.release_date))`.
Then return the needed format using the Date object methods: `return releaseDate.toLocaleString('en', { dateStyle: 'long' })`

File: src/views/MoviePageView.vue [hunk 6]
```diff
@@ -0,0 +1,153 @@
...
+  },
+  computed: {
+    item() {
+      return this.$store.getters['movie/getItem']
+    },
+    credits() {
+      return this.$store.getters['movie/getActors']
+    },
+    posterUrl() {
+      return `https://image.tmdb.org/t/p/original${this.item.poster_path}`
```
Comment (skss-1) [inline src/views/MoviePageView.vue]: `https://image.tmdb.org/t/p/original` already stored in the `tmdb-api.js` file. Please use it from there.
*Note:* the same for 'backgroundUrl'

File: src/router/index.js [hunk 7]
```diff
@@ -10,6 +11,11 @@ const routes = [
     name: 'home',
     component: HomeView
   },
+  {
+    path: '/movie/:id',
+    name: 'movie-page',
+    component: MoviePageView,
+  },
   {
     path: '/about',
```
Comment (skss-1) [inline src/router/index.js]: Please, remove this route and `AboutView` because it is redundant

File: src/router/index.js [hunk 8]
```diff
@@ -10,6 +11,11 @@ const routes = [
     name: 'home',
     component: HomeView
   },
+  {
+    path: '/movie/:id',
+    name: 'movie-page',
+    component: MoviePageView,
```
Comment (skss-1) [inline src/router/index.js]: It's not required for now, but better to use the `import()` functions for routes. As example you can see how it made for `/about` route

File: src/components/MovieCard.vue [hunk 9]
```diff
@@ -1,6 +1,9 @@
...
+    @click="goToMoviePage"
```
Comment (skss-1) [inline src/components/MovieCard.vue]: For better navigation, I suggest to add `tabindex="0"` for the root element. Also add event handlers for `@keydown.enter.esc="goToMoviePage"`. User will have possibility to navigate through the cards via keyboard

**Review by skss-1 — CHANGES_REQUESTED** [review]

File: src/views/MoviePageView.vue (hunk 5 above)
Comment (nikitius20) [reply src/views/MoviePageView.vue]: Thx, I didn't even knew about this

**Review by skss-1 — APPROVED** [review]

**Pull request merged**